### PR TITLE
Remove Webhook Subcription URL

### PR DESCRIPTION
**Pull request by hydra-yse**
Closes breez/breez-sdk#730

This PR aims at giving the user the possibility to unsubscribe to a given notification by providing its associated webhook's URL.

### CODE REVIEW

**Review by roeierez — APPROVED** [review]
LGTM

File: notifications/server.go [hunk 1]
```diff
@@ -82,3 +82,59 @@ func (s *server) SubscribeNotifications(
 	log.Printf("%v was successfully registered for notifications on url %s", pubkey, request.Url)
 	return &SubscribeNotificationsReply{}, nil
 }
+
+func (s *server) UnsubscribeNotification(
```
Comment (JssDWt) [inline notifications/server.go:86]: This function in not in the proto service definition. There's 2 options I think:
1) Add it to the service definition. If we do that, we need a PR in breez server as well.
2) reuse the same encrypted request on SubscribeNotifications and check which function is being called by decrypting the payload.

**Review by JssDWt — COMMENTED** [review]
Looks good!
Maybe we can add a tiny integration test for the subscribe/unsubscribe by only running lspd and checking the postgres database in the test?

**Review by JssDWt — APPROVED** [review]
LGTM!

**Pull request merged**